mne_bids/meg_bids.py

```diff
@@ -116,7 +111,7 @@ def _scans_tsv(raw, raw_fname, fname, verbose):
         acq_time = datetime.fromtimestamp(
             meas_date).strftime('%Y-%m-%dT%H:%M:%S')
 
-    df = pd.DataFrame({'filename': ['%s' % raw_fname],
+    df = pd.DataFrame({'filename': ['%s' % raw_file],
```

Comment:
why change this? Don't we use `fname` as the convention for filename everywhere in MNE?